aspnetcore/mvc/views/tag-helpers/builtin/resources/AnchorTagHelper.md

```diff
@@ -93,7 +89,9 @@ When the link is clicked, this will call the controller method ```EvaluationsCur
 asp-all-route-data``` 
 dictionary.
 
-## asp-fragment
+- - -
+
+### asp-fragment
 
 ```asp-fragment``` appends after a hash (```#```) tag at the end of the URL whatever the value assigned to it is. That is, if you create a tag
```

Comment:
> ```asp-fragment``` appends after a hash (```#```) tag at the end of the URL whatever the value assigned to it is. That is, if you create a tag

 ```asp-fragment``` defines a URL fragment to append to the URL. The Anchor Tag Helper will add the hash character (`#`) automatically. If you create a tag: